gatsby/src/components/Interaction/Interaction.module.scss

```diff
@@ -0,0 +1,19 @@
+@use '@nice-digital/nds-core/scss/core';
+
+.messageList {
+  list-style: none;
+  margin: 0;
+}
+
+.interactantTitle {
+  @include core.nds-h6();
+}
+
+.message + .message {
+  border-top: 2px dashed #828282; // TODO: Reconcile with the design system
+}
+
+.severeMessage {
+  border-left: 6px solid #e10303; // TODO: Reconcile with the design system
```

Comment:
Something like:
```suggestion
  border-left: core.rem(core.$nds-spacing-small) solid core.$nds-colour-error;
```

Reply:
Thanks - one issue here is that the colours used in the interaction designs don't match anything already in the design system, so I need to catch up with Nick to see whether we should:

- add more colours to the system,
- add them to the BNF site only, or 
- just use existing design system colours instead

Reply:
For now I've used the error colour, but we'll need to add a new variable to the design system for a warning red.